File: C++/1440.cpp
#include <iostream>

using namespace std;

int main(){
	int n;
	cin >> n;
	int num[n];
	char oper[n];
	
	for(int i = 0; i < n; i++){
		cin >> num[i];
	}
	
	for(int i = 0; i < n; i++){
		printf("%d: ", i+1);
		for(int j = 0; j < n; j++){
			if(i!=j){
				if(num[i] > num[j])
					printf("> ");
				else if(num[i] == num[j])
					printf("= ");
				else
					printf("< ");
			}
		}
		cout << endl;
	}
}

File: Java/1440.java
import java.util.Scanner;
import java.util.StringTokenizer;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int num[] = new int[n];
		char oper[] = new char[n];
		
		for(int i = 0; i < n; i++) {
			num[i] = sc.nextInt();
		}
		
		for(int i = 0; i < n; i++) {
			System.out.printf("%d: ", i+1);
			for(int j = 0; j < n; j++) {
				if(i!=j) {
					if(num[i] > num[j])
						System.out.print("> ");
					else if(num[i] == num[j])
						System.out.print("= ");
					else
						System.out.print("< ");
				}
			}
			System.out.println();
		}
	}
}